system-helper/flatpak-system-helper.c

```diff
@@ -1201,12 +1201,13 @@ handle_update_summary (FlatpakSystemHelper   *object,
  * allows authorization decisions to be made based on them, without hard-coding
  * a policy here. */
 static gboolean
-authorize_deploy_add_polkit_details (const gchar    *installation,
-                                     const gchar    *ref,
-                                     const gchar    *origin,
-                                     PolkitSubject  *subject,
-                                     PolkitDetails  *details,
-                                     GError        **error)
+authorize_deploy_add_polkit_details (const gchar           *installation,
+                                     GDBusMethodInvocation *invocation,
```

Comment:
This “system-helper: Properly use dir_get_system()” commit should have something in its commit message saying that it needs to be squashed into “system-helper: Move parental controls checks to system-helper” on the next rebase.